Site/BSSwFellowshipProgram/People/SukhijaNi.md

```diff
@@ -10,14 +10,19 @@
+The sharp increase in computational power of computing ecosystems is likely to continue as we move toward exascale and beyond.  In turn, we are seeing new convergent computing platforms along with a paradigm shift in scientific software applications leveraging these platforms. Unfortunately, this also leads to an unexpected growth in security risks pertaining to cybercriminals, as well as malicious insiders in the computing ecosystems. To  address these issues, it is of paramount importance to integrate security within the scientific software development lifecycle. The need for best practices for secure software development has been highlighted in the President’s Executive Order on Improving the Nation’s Cybersecurity issued in May, 2021.
 
 Nitin Sukhija will create a one-day workshop on securing scientific software development. The workshop components will include evaluating design practices for creating secure software, software processes for managing secure software, threat modeling, and quality assurance testing using both static and dynamic analysis tools.  The workshop will include hands-on exercises with penetration testing tools and how to mitigate threats such as losing sensitive information due to a variety of potential vulnerabilities.  The workshop is intended to aid members from diverse research domains in development of trustworthy and secure scientific software.
 
 Nitin Sukhija is an associate professor in the department of computer science and director of Center for Cybersecurity and Advanced Computing (C2AC) at Slippery Rock University of Pennsylvania. His areas of expertise are scientific computing focusing on performance modeling, robustness and resilience analysis, and cybersecurity.
+
+### Selected Resources
+<a href="https://bssw.io/blog_posts/secure-software-programming-practices-and-development" class="link-row">Secure Software Programming Practices and Development blog article</a>
+<a href="https://zenodo.org/communities/secure-scientific-software-development/" class="link-row">Best Practices and Tools for Secure Scientific Software Development</a>
+<a href="https://ideas-productivity.org/events/hpcbp-081-securesoftware" class="link-row">HPC Best Practices Webinar - Secure Software Programming Practices and Development</a>
```

Comment:
Actually, looking at the rendered page [here](https://github.com/gonsie/bssw.io/blob/bsswf22-updates/Site/BSSwFellowshipProgram/People/SukhijaNi.md), I think we want to provide bullets for these references.  Right now they just all run together.

Reply:
Hi @bartlettroscoe, I think the CSS and HTML class "link-row" will do that for us. See:
- Full render: https://bssw.io/fellows/julia-stewart-lowndes
- Source: https://github.com/gonsie/bssw.io/blob/bsswf22-updates/Site/BSSwFellowshipProgram/People/LowndesJu.md

Reply:
@gonsie, I just rebuilt the preview site with this branch merged to the 'preview' branch and you are correct.  As shown at:

* https://preview.bssw.io/fellows/nitin-sukhija

and it shows:

![image](https://github.com/betterscientificsoftware/bssw.io/assets/1959736/eb9fc426-fa34-4d6a-bbfe-235ef5dfab78)

But is is unfortunate that the direct Markdown rendering of these pages like in GitHub does not show this structure ☹️